projects/ng-components-docs/tsconfig.app.json

```diff
@@ -11,5 +11,6 @@
             "path": ["lib/src/button/*"]
         }
     },
-    "exclude": ["test.ts", "**/*.spec.ts"]
+    "files": ["src/main.ts", "src/polyfills.ts"],
```

Comment:
is it ok to remove exclude ?

Reply:
this was done automatically by ng update script. looks like they are moving away from exclude to include or files instead. But to be honest I'm not sure. I need to test out all the build scripts etc